src/key.js

```diff
@@ -157,6 +160,16 @@ Key.prototype.toPacketlist = function() {
   return packetlist;
 };
 
+/**
+ * Throws error if signature has revocation key sub packet (not supported by OpenPGP.js)
+ */
+Key.prototype.checkRevocationKey = function(signature) {
+  if (signature.revocationKeyClass !== null &&
+      signature.issuerKeyId.equals(this.getKeyId())) {
```

Comment:
You have `this.getKeyId()` available as `primaryKeyId` in the calling locations, you could pass that instead of using `this` if you like